bootstrap.py

```diff
@@ -156,6 +156,9 @@ def install_prereqs():
     yum("remove", "subscription-manager-gnome")
     yum("install", "subscription-manager 'subscription-manager-migration-*'")
     yum("update", "yum openssl python")
+    # > Clean the environment from LD_... variables
+    clean_environment()
```

Comment:
this is too late. clean_environment is there to fix up unclean environments as created by BMC BladeLogic. those tend to break yum. I'd argue that we should instead split out the fact generation out of clean_environement, and call that part here instead.